h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -24,13 +21,21 @@
 import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Consumer;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
 @Ignore("Support for tests, but no actual tests here")
 public class TestUtil extends Iced {
+  {
```

Comment:
What is the motivation to add this?

I think assertions should be enabled for all H2O instances used in tests. This will enabled assertions only in the JVM that is driving the tests - is this intended?